docs/api_guide/plot_06_sklearn_pipeline_cv_demo.py

```diff
@@ -209,6 +219,15 @@
 # %%
 # #### Run the grid search
 # Let's run a 5-fold cross-validation of the hyperparameters with the scikit-learn `model_selection.GridsearchCV` class.
+# ??? info "K-Fold cross-validation (click to expand/collapse)"
+#     <p align="center">
+#     <img src="../../../assets/grid_search_cross_validation.png" alt="Grid Search Cross Validation" style="max-width: 80%; height: auto;">
```

Comment:
this image is very slightly trimmed (the final "s" in "Finding parameters"). Can probably be fixed by sliding over the text label a bit.

And probably worth including this file as an svg? in order to make it easier to edit

Reply:
The original png of sklearn was, i will create my own version as svg